lib/config/presets/internal/replacements.ts

```diff
@@ -901,6 +902,18 @@ export const presets: Record<string, Preset> = {
       },
     ],
   },
+  'standard-version-to-maintained': {
```

Comment:
```suggestion
  'standard-version-to-commit-and-tag': {
```

This is more descriptive to me. It also matches the style of other similar presets.